Assets/Scripts/Controllers/GroundController.cs

```diff
@@ -0,0 +1,52 @@
+using Ground;
+using UnityEngine;
+
+namespace Controllers
+{
+    /// <summary>
+    /// Sits in the world, generating new ground objects when the previous one has moved enough distance
+    /// </summary>
+    ///
+    public class GroundController : MonoBehaviour
+    {
+        // Start is called before the first frame update
```

Comment:
[Minor] This comment is way above where it originally was and doesn't at all apply to the fields